src/engine/PiecewiseLinearConstraint.cpp

```diff
@@ -201,6 +204,35 @@ void PiecewiseLinearConstraint::setStatistics( Statistics *statistics )
     _statistics = statistics;
 }
 
+bool PiecewiseLinearConstraint::phaseFixed() const
```

Comment:
When _cdPhaseStatus is not null, does this method always have the same output as isImplication()? If so do we want to merge the two methods?

Reply:
Longterm we want to merge the two methods. This PR already does a lot and I didn't want to complicate things further, since I am not convinced that this would be a trivial change (e.g. Max/Disjunction semantics). PhaseFixed currently is derived from bound propagation while isImplication is derived from search/backtracking.